sdks/python/apache_beam/testing/analyzers/load_test_perf_analysis.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+from apache_beam.testing.analyzers import constants
+from apache_beam.testing.analyzers import perf_analysis_utils
+from apache_beam.testing.analyzers import perf_analysis
+from apache_beam.testing.analyzers.perf_analysis_utils import MetricContainer, TestConfigContainer
+from google.cloud import bigquery
```

Comment:
google.cloud dependencies are optional for beam python SDK. Use this pattern to avoid beam fail to import when `[gcp]` extension not installed: 

https://github.com/apache/beam/blob/9e4668d725fdcc5683541499b0d9e45f9b0ef71d/sdks/python/apache_beam/io/gcp/bigtableio.py#L58-L59

Reply:
Done